src/main/java/frc/robot/PID.java

```diff
@@ -0,0 +1,36 @@
+package frc.robot;
+
+
+
+public class PID {
+    
+    private double Kp, Ki,I, Kd, error, lasterror, timeIncrement, time, timeremaining, lastSensorRead, decreaseSpeed, pastPoint;
+
+    public PID(double Kp, double Ki, double Kd, double time, double lastSensorRead){
+    this.Kd = Kd;
+    this.Ki = Ki;
+    this.Kp = Kp;
+    this.time = time;
+    this.lastSensorRead = lastSensorRead;
+    timeremaining = time*50;
+    I = 0;
```

Comment:
Ki, Kp, and Kd should probably be defined. They should be camel case

a variable named "I" is something you definitely should consider changing